docs/conf.py

```diff
@@ -67,6 +67,8 @@
     ("py:class", "torchvision.models._api.WeightsEnum"),
     ("py:class", "torchvision.models.resnet.ResNet"),
     ("py:class", "torchvision.models.swin_transformer.SwinTransformer"),
+    ("py:class", "fiona.Feature"),
+    ("py:class", "fiona.model.Feature"),
```

Comment:
Since we don't refer to Feature in the codebase yet I changed our type annotations to use `fiona.model.Feature` which obviates the need for including `fiona.Feature` here. And it is alphabetical now.